### PR TITLE
Make sync logging a bit less verbose

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

The sync logging is a bit verbose, let's fix that.

### CODE REVIEW

**Comment by teor2345** [comment]
This belongs in sprint 3, because it's a cleanup after a recent PR.

File: zebrad/src/components/sync.rs [hunk 1]
```diff
@@ -351,7 +351,7 @@ where
             })
             .map_err(|e| eyre!(e))?;
 
-        tracing::info!("trying to obtain new chain tips");
+        tracing::info!(?tip = block_locator.first().unwrap(), "trying to obtain new chain tips");
```
Comment (teor2345) [inline zebrad/src/components/sync.rs]: Oops
```suggestion
        tracing::info!(tip = ?block_locator.first().unwrap(), "trying to obtain new chain tips");
```

File: zebrad/src/components/sync.rs [hunk 2]
```diff
@@ -466,8 +466,9 @@ where
         let tips = std::mem::take(&mut self.prospective_tips);
 
         let mut download_set = HashSet::new();
+        tracing::info!(?tips = tips.len(), "trying to extend chain tips");
```
Comment (teor2345) [inline zebrad/src/components/sync.rs]:
```suggestion
        tracing::info!(tips = ?tips.len(), "trying to extend chain tips");
```

**Comment by teor2345** [comment]
The windows testnet CI failure will be fixed next sprint by #1222

**Comment by teor2345** [comment]
The CI failure is #1730.

**Review by dconnolly — APPROVED** [review]

**Pull request merged**